osbenchmark/worker_coordinator/worker_coordinator.py

```diff
@@ -293,8 +529,19 @@ def receiveMsg_WakeupMessage(self, msg, sender):
     def create_client(self, host):
         return self.createActor(Worker, targetActorRequirements=self._requirements(host))
 
-    def start_worker(self, worker_coordinator, worker_id, cfg, workload, allocations):
-        self.send(worker_coordinator, StartWorker(worker_id, cfg, workload, allocations))
+    def start_worker(self, worker_coordinator, worker_id, cfg, workload, allocations, error_queue=None, queue_lock=None):
```

Comment:
same here, lets preserve the original behavior and add the new behavior if config is set.